packages/app-desktop/gui/DefaultEditorDialog/Dialog.tsx

```diff
@@ -0,0 +1,231 @@
+import * as React from 'react';
+import { useRef, useCallback } from 'react';
+import { _ } from '@joplin/lib/locale';
+import DialogButtonRow from '../DialogButtonRow';
+import Dialog from '../Dialog';
+import styled from 'styled-components';
+import DialogTitle from '../DialogTitle';
+import useElementSize from '@joplin/lib/hooks/useElementSize';
+import Button, { ButtonLevel } from '../Button/Button';
+import bridge from '../../services/bridge';
+import Setting from '@joplin/lib/models/Setting';
+
+interface Props {
+	themeId: number;
+	// eslint-disable-next-line @typescript-eslint/ban-types -- Old code before rule was applied
+	onClose: Function;
```

Comment:
```suggestion
	onClose: ()=>void;
```

Avoid disabling ESLint rules where possible.